oauth2-useragent-javafx/src/main/java/com/microsoft/alm/oauth2/useragent/InterceptingBrowser.java

```diff
@@ -113,6 +115,10 @@ static boolean matchesRedirection(final String expectedRedirectUriString, final
             }
         }
 
+        if (actualUri.getScheme().equals("urn") && !actualUriString.startsWith(expectedRedirectUriString)) {

```

Comment:
You wrote earlier that using `URI.create()` on a string like `urn:ietf:wg:oauth:2.0:oob?code=abc&stat=123` did not accurately represent the input.  Should we do the same thing in the `matchesRedirection()` method instead of adding a scheme-specific check?